heap/heap.py

```diff
@@ -1,21 +1,55 @@
 class Heap:
-  def __init__(self):
-    self.storage = []
+    def __init__(self):
+        self.storage = []
 
-  def insert(self, value):
-    pass
+    def insert(self, value):
+        self.storage.append(value)
+        self._bubble_up(len(self.storage)-1)
 
-  def delete(self):
-    pass
+    def delete(self):
+        heap_top = self.storage[0]
+        heap_end = self.storage[-1]
+        if (self.get_size()>1):
+          self.storage = [heap_end] + self.storage[1:-1]
```

Comment:
This looks great! Using self.storage.pop() is another way to remove the last element in the heap.